tests/features/monrun.feature

```diff
@@ -139,6 +139,23 @@ Feature: ch-monitoring tool
     """
     0;OK
     """
+    When we execute query on clickhouse01
+    """
+    SYSTEM STOP FETCHES
+    """
+    And we execute query on clickhouse02
+    """
+    INSERT INTO test.table_01 SELECT number FROM numbers(100)
+    """
+    And we sleep for 5 seconds
+    And we execute command on clickhouse01
+    """
+    ch-monitoring replication-lag -w 4
+    """
+    Then we get response contains
+    """
+    1;
+    """
```

Comment:
I was thinking about the same test that detects replication lag for `replication-sync` command, but default timeout for `replication-lag` is 300 seconds. This would require to sleep for 300 seconds and will slow tests a lot.
Maybe we should pass args for `replication-lag` to `replication-sync` as discussed above to avoid this

Reply:
I think it makes sense